enhancements/network/core-network-policies.md

```diff
@@ -0,0 +1,547 @@
+        
+    3. ACS can also be used to analyze a folder with all of the
+       namespace YAML resources to heuristically try to figure out who is
+       trying to talk to whom and mint tight network policies to reflect
+       that model.
+    
+    4. The Network Observability tool can be used to detect when a network policy blocked traffic
+    
+3. Auditing policies:
+
+    5. Someone familiar with network policies should work with the
+       team to ensure that the policies are as tight as is possible given
+       the expected traffic
+    
+4. Deploying policies:
+
+    6. The network policies need to be deployed by the operator in to the operand namespaces
+
+
+### API Extensions
+
+This extends the API server by changing the Admission Controller on
+namespace objects so that the label `security.openshift.io/openshift-namespace` is applied
+or removed depending on whether the namespace name starts with
+`openshift-`.
+
+Other than that, there is no other API change.
+
+
+### Topology Considerations
+
+
+#### Hypershift / Hosted Control Planes
+
+Are there any unique considerations for making this change work with Hypershift?
+
+No. 
+
+
+#### Standalone Clusters
+
+Is the change relevant for standalone clusters?
+
+No.
+
+
+#### Single-node Deployments or MicroShift
+
+How does this proposal affect the resource consumption of a
+single-node OpenShift deployment (SNO), CPU and memory?
+
+No change, other than the creation of some additional objects in the API server.
+
+How does this proposal affect MicroShift? For example, if the proposal
+adds configuration options through API resources, should any of those
+behaviors also be exposed to MicroShift admins through the
+configuration file for MicroShift?
+
+No impact.
+
+
+### Implementation Details/Notes/Constraints
+
+What are some important details that didn't come across above in the
+**Proposal**? Go in to as much detail as necessary here. This might be
+a good place to talk about core concepts and how they relate. While it
+is useful to go into the details of the code changes required, it is
+not necessary to show how the code will be rewritten in the
+enhancement.
+
+
+#### DNS
+
+Almost every pod will need to have an egress policy to allow DNS.
+Would it be better to have one policy that can be enabled by a label?
+Or just have an AdminNetworkPolicy that just allows it.
+
+
+#### Host Network Pods
+
+Network Policy does not apply to pods that are running in the host
+network.  However, other pods that expect to receive connections from
+the host network on nodes will need
+[policy](https://docs.openshift.com/container-platform/4.17/networking/network_security/network_policy/about-network-policy.html)
+to allow it.
+
+
+#### API Server
+
+The API servers have to be specified by IP address, labels do not
+work.  We need a policy that is maintained by an operator that has a
+well-known label that pods can have applied to them.
+
+
+#### Monitoring
```

Comment:
It might not be directly related to this EP but do we have metrics reporting the number of flows being blocked by network policy?